### ISSUE TITLE
learning rate not updating in finetetuning

### ISSUE THREAD

**Issue by Shahidul1004** [xinntao/Real-ESRGAN#97]
I am getting lr=2e-4 all the time. I tried to update `options/finetune_realesrgan_x4plus_pairdata.yml` but still the lr not updating

**Comment by xinntao** [comment]
I have tested on my side. It is OK. 

You may have several folders for Real-ESRGAN?
